test/.eslintrc

```diff
@@ -1,7 +1,15 @@
-// Use this file as a starting point for your project's .eslintrc.
-// Copy this file, and add rule overrides as needed.
 {
-  "extends": ["@easypost/eslint-config-easypost-base"],
+  "extends": [
+    "@easypost/eslint-config-easypost-base",
+    "prettier",
+    "prettier/@typescript-eslint",
+    "prettier/babel",
+    "prettier/flowtype",
+    "prettier/react",
+    "prettier/standard",
+    "prettier/unicorn",
+    "prettier/vue"
+  ],
 
```

Comment:
I think we have an extra line here

Reply:
Yeah, each of these sections is separated by a line.